parts/3-clinical/slides.md

```diff
@@ -1,108 +1,113 @@
 weeks*.
 
-So why are people still hesitant?
-
-1. The high initial cost. One challenge is to prove that sequencing can
-   effetively replace many of the old inconclusive tests.
-
-2. Analysis of data is neither trivial nor standardized. For example how
-   do you distingish an error from a mutation? Essentially 99.9% of the
-   data generated is either too difficult to interpret or completely
-   non-informative. Most sequences are just the same as what we know we
-   find healthy individuals. To be honest we are generating a lot of
-   "waste" that no ones has figured out a plan to do with. Kind of like
-   a nuclear power plant.
-
+## How?
 As the patient sample travels through our facility thousands of lines of
 Python code will eventually touch it. I'm going to focus on the delivery
 report.
 
+This is the story...
+
 
 ## Excel sheet -> Web interface
-What these guys [point to Guillermo] are delivering to researchers is very
-basic text files usually. Low level data that needs a lot of processing to
-start making sense. We have the ambition to take things a lot further.
-Basically we want to eliminate the need for any further non-manual
+In the beginning there was ... Excel. At least it's something familiar to
+most people. Soon enough it got out of hand. It was very inflexible to
+work with. Thousands of rows in an unwieldy and inherently static grid.
+
+Researchers you can give very basic text files that need a lot of processing to
+start making sense. We want to eliminate the need for any further non-manual
 processing after we deliver our results.
 
-We also don't focus much on providing extensive tools to filter out the
-interesting data but more on prioritizing what they should look into first.
-This is meant to be part of routine work and not exploratory research so
-it makes sense.
 
-In the beginning there was ... Excel. At least it's something familiar to
-most people. Soon enough it got out of hand. It was very inflexible to
-work with. Thousands of rows in an unwieldy, and inherently static grid.
-It also took too much time to look into each case to be practical.
+## The Scout interface
+To reduce the time it takes to go over each case we focus a lot of prioritizing the
+information. This helps doctors to know what they should look at first. This is meant
+to be part of routine work and not exploratory research so it makes sense.
+
+
+## JavaScript vs. Python code
+And then there was ... lots of JavaScript. But our input isn't optimized for this
+kind of setup: we have mostly non-updating data and big tables which gives for
+sluggish client side performance.
+
+I can confess that when I started this project I saw it as a great opportunity to
+learn more about e.g. Ember.js so I just picked it for the project. Only after my geek
+entuthiasm had died down did I realize that this is a professional, clinical grade tool
+and we shouldn't concern ourselves with user appeal too much.
```

Comment:
I think there's a compliment in there so I'll start by saying; thanks!

Also you can always make things pretty :blush: but still try to avoid actual complexity to the code by say rendering everything in javascript because you ... feel like it...